app/views/partials/attachments_list_atar.scala.html

```diff
@@ -0,0 +1,93 @@
+        implicit request: AuthenticatedRequest[_], messages: Messages)
+
+
+@if(attachments.nonEmpty){
+        <table id="@{id}-table" class="govuk-table govuk-!-font-size-16">
+            <thead class="govuk-table__head">
+                <tr>
+                    <th class="govuk-table__header" scope="col">File name</th>
+                    <th id="@{id}-header-uploaded_by" class="govuk-table__header" scope="col">Added by</th>
+                    <th id="attachment-details" class="govuk-table__header" scope="col">Details</th>
+                    <th id="attachment-status" class="govuk-table__header" scope="col">Status</th>
+                    <th><span class="visually-hidden">attachment action</span></th>
+                </tr>
+            </thead>
+            <tbody class="govuk-table__body">
+                    @for((file: StoredAttachment, index: Int) <- attachments.zipWithIndex) {
+                        <tr id="@{id}-row-@{index}" class="govuk-table__row">
+                            <span class="pre-wrap"></span>
+                            <td id="@{id}-row-@{index}-title" class="govuk-table__cell">
+                                @attachment_atar(s"$id-row-$index", file, useTabIndex = startAtTabIndex.map(x => x+1+(0*index)))
+                            </td>
+                        </span>
+                                <td id="@{id}-row-@{index}-uploaded_by" class="govuk-table__cell w-15">
+                                    @if(file.operator.isDefined) {
+                                        @file.operator.flatMap(_.name).getOrElse("Unknown")
+                                    } else {
+                                    @c.application.contact.name
+                                    }
+                                </td>
+
+                            <td id="@{id}-row-@{index}-attachment-details" class="govuk-table__cell w-15">
+                                @if(file.description.isDefined) {
+                                     @file.description
+                                }
+                            </td>
+
+                            <td id="@{id}-row-@{index}-attachment-status" class="govuk-table__cell w-15">
+                                <span class="visually-hidden">attachment-status</span>
+                                @if(file.scanStatus.contains(ScanStatus.READY)) {
+                                  @{(file.public, file.shouldPublishToRulings) match {
+                                          case (false, _) =>
+                                              <span id="@{id}" class="govuk-tag govuk-tag--yellow"> {messages("case.attachment.upload.status-confidential")}</span>
+                                          case (true, true) =>
+                                              <span id="@{id}" class="govuk-tag govuk-tag--green"> {messages("case.attachment.upload.status-published")}</span>
```

Comment:
I think that `published` in the classification service used to mean that the virus scan had completed (`file.scanStatus.contains(ScanStatus.READY)`), and that `shouldPublishToRulings` only controlled whether it appeared on the rulings website when the case was completed, has that changed as part of this ticket?

Reply:
No, it has not changed. However, the Attachments tab in the prototype contains  the Status "PUBLISHED" .